packages/components/.babelrc

```diff
@@ -1,14 +1,14 @@
 {
   "presets": [
     "es2015",
-    "react"
+    "react",
```

Comment:
are you sure this is working in RC file ?
because AFAIK this is json, and this is not a valid json